functions/pipeline/shared/db_access_v2.py

```diff
@@ -6,7 +6,7 @@
 from enum import IntEnum, unique
 import getpass
 import itertools
-from db_provider import DatabaseInfo, PostGresProvider
+from .db_provider import DatabaseInfo, PostGresProvider
```

Comment:
pytest failure might be caused by this change here. Currently working on some changes to fix some importing bugs and structure our /shared directory.

Reply:
https://github.com/mtarng/active-learning-detect/tree/json-processing-no-creds-rebase

Working branch for those changes right now if you want a reference.